pkg/cmd/config.go

```diff
@@ -871,7 +877,8 @@ addons:
 # The API server can also be overwritten by Terraform. Provide the
 # external address of your load balancer or the public addresses of
 # the first control plane nodes.
-# apiEndpoint:
+apiEndpoint:
+  alternativeNames: {{ .APIEndpointAlternativeNames }}
```

Comment:
Can we have it commented like host and port? Also, very tiny nit, but I think it might make sense to put this after host and port because those two fields are more important.

Reply:
Done